src/components/DateRangePicker.jsx

```diff
@@ -248,7 +248,7 @@ class DateRangePicker extends React.PureComponent {
     //
     // We handle both situations here by using the ` || ` operator to fallback
     // to *event.target** when **relatedTarget** is not provided.
-    if (this.dayPickerContainer.contains(event.relatedTarget || event.target)) return;
+    if (this.dayPickerContainer.contains(event.relatedTarget || event.target) || this.dayPickerContainer.contains(event.target)) return;
```

Comment:
If possible, can you add a test here to cover this?

In IE 11, what element does it provide as the `relatedTarget` when you actually want the `target`?

Reply:
In IE11 relatedTarget is <body>, so the condition is false and all clicks are closing the datepicker

Reply:
Would it be more targeted to make the current `||` instead be something like `event.relatedTarget === document.body ? event.target : event.relatedTarget || target`?